package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "node.mentoring.program.2018",
+  "version": "1.0.0",
+  "description": "NodeJS Mentoring program",
+  "main": "index.js",
+  "repository": "https://github.com/kuzanatoliy/node.mentoring.program.2018.git",
+  "author": "Anatoli Kuzmiankou <Anatoli_Kuzmiankou@epam.com>",
+  "license": "MIT",
+  "scripts": {
+    "lint": "eslint . --ext .js",
+    "dev": "nodemon index.js",
+    "start": "node index.js",
+    "test": "test"
+  },
+  "devDependencies": {
+    "babel-core": "6.26.0",
+    "babel-preset-env": "1.6.1",
+    "babel-preset-stage-2": "6.24.1",
+    "babel-register": "6.26.0",
```

Comment:
Instead of babel-register you can set up babel-cli. So then you can run your application with the command _babel-node app.js_ and remove _babel-register_ as far as _babel-core_ (there won't be cases when it is necessary to use API). Also the replacement will help get rid of _index.js_ (there is no necessity to keep that file at all since the main file should be _app.js_),

Reply:
But babel-node working whithout chenging package.

Reply:
Speaking about modules, that's a good point. Well, all the modules we are going to use are supposed to be transpiled. If they don't (that is unlikely) we can get those changes back. Btw, all requires to node_modules using babel-register are ignored by default: https://babeljs.io/docs/usage/babel-register/#ignores-node_modules-by-default.